hack/go2md/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+go build .
+
+./go2md /root/code/github.com/talos-systems/talos/pkg/config/types/v1alpha1/machine_config.go /root/code/github.com/talos-systems/talos/docs/website/content/v0.3/en/configuration/v1alpha1.md
```

Comment:
I wonder if we can add this to a go generate block.. maybe under `pkg/config/types/v1alpha1/docs.go`?

not sure if this is entirely right, but im sure you get the gist
```
//go:generate -command go run ../../../../hack/go2md/main.go ../../../../docs/website/content/v0.3/en/configuration/v1alpha1.md
```

Reply:
done